docs/FAQ.md

```diff
@@ -17,3 +17,11 @@ Here we discuss frequently asked questions that may occur and we found useful al
 4. Can I add custom datasets?
 
     Yes, you can find more information on how to do that [here](Dataset.md).
+
+5. What are the hardware SKU requirements for deploying these models?
+
+    Hardware requirements vary based on latency, throughput and cost constraints. For good latency, the models were split across multiple GPUs with tensor parallelism in a machine with NVIDIA A100s or H100s. But TPUs, other types of GPUs, or even commodity hardware can also be used to deploy these models (e.g. https://github.com/ggerganov/llama.cpp).
```

Comment:
We should add Inference optimized SKUs like Nvidia A10g, T4, L4. 

For CPU side will be good to link the Intel blog for [Accelerating Llama 2 on Intel hardware / software](https://www.intel.com/content/www/us/en/developer/articles/news/llama2.html)